src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -86,6 +86,14 @@ public function createListFromChoices($choices, $value = null)
     {
         if (is_string($value) && !is_callable($value)) {
             $value = new PropertyPath($value);
+        } elseif (is_string($value) && is_callable($value)) {
+            @trigger_error('Treating strings as callable is deprecated since version 3.1 and will throw an error in 4.0. You should use a "\Closure" instead.', E_USER_DEPRECATED);
+            
+            $value = function (array $choices) use ($value) {
+                return $value($choices);
+            };
+
+            return $value($choices);
```

Comment:
I've followed the instruction of someone since I don't wanted to make things bad,
But you're right, adding the test without any code is the right thing to do.

Thanks